NOTES.rst

```diff
@@ -1687,7 +1687,7 @@ The hierarchy/naming conventions in the ``alertmsg/`` directory are as follows::
 
 The `<event type>` is one of the available event types in NAV, whereas `<alert
 type>` is one of the alert types associated with the event type. `<medium>` is
-one of the supported alert mediums, such as `email`, `sms` or `jabber`. A two
+one of the supported alert mediums, such as `email`, `sms` or `slack`. A two
```

Comment:
IMHO, it is blatantly wrong to change the historical release notes for the 3.13 release. At this point, NAV **did** support Jabber, but **not** Slack.

Reply:
I obviously did not read enough context when changing this. My bad